src/domain/interfaces/balances-api.interface.ts

```diff
@@ -5,7 +5,7 @@ import { Page } from '@/domain/entities/page.entity';
 export interface IBalancesApi {
   getBalances(args: {
     safeAddress: string;
-    fiatCode?: string;
+    fiatCode: string;
```

Comment:
`fiatCode` is not optional anymore, both `ZerionBalancesApi` and `SafeBalancesApi` need a `fiatCode` parameter. On the old implementation `SafeBalancesApi` did not need this parameter as the fiat amounts were calculated in a further step, but now all the logic is pushed into the data sources layer.